integration-scripts/src/deployV0.6Contracts.ts

```diff
@@ -25,7 +25,7 @@ export async function deployContracts() {
     { Factory: FluxAggregatorFactory, signer, name: 'FluxAggregator' },
     linkToken.address,
     1,
-    3,
+    60,
```

Comment:
What is this change about?

Reply:
This raises the round timeout to 60 seconds so that we get more deterministic test behavior, particularly if the CI node happens to be under load.